index.js

```diff
@@ -137,9 +137,14 @@ alfy.fetch = async (url, options) => {
 		return cachedResponse;
 	}
 
+	const rawBodyRequested = 'json' in options && options.json === false;
+	if (rawBodyRequested) {
+		delete options.json;
+	}
+
 	let response;
 	try {
-		response = await got(url, options).json();
+		response = rawBodyRequested ? (await got(url, options)).body : await got(url, options).json();
```

Comment:
Instead of using the `.json()` option, you could set the option instead: https://github.com/sindresorhus/got/blob/main/documentation/2-options.md#responsetype

Then you don't need two `got` calls.

Reply:
Updated - and since it's dealing with `options` objects instead of the response method, moved it together with other option manipulating blocks